src/libraries/Common/src/Interop/Interop.Ldap.cs

```diff
@@ -99,6 +99,7 @@ internal enum LdapOption
         LDAP_OPT_SECURITY_CONTEXT = 0x99,
         LDAP_OPT_ROOTDSE_CACHE = 0x9a, // Not Supported in Linux
         LDAP_OPT_DEBUG_LEVEL = 0x5001,
+        LDAP_OPT_URI = 0x5006,
```

Comment:
I suppose this is not supported on Windows, if so could we add a comment saying so?

Reply:
Sure thing.